visualSHARK/models.py

```diff
@@ -190,4 +190,29 @@ class IssueValidation(models.Model):
 class IssueValidationUser(models.Model):
     user = models.ForeignKey(User, on_delete=models.CASCADE)
     issue_validation = models.ForeignKey(IssueValidation, on_delete=models.CASCADE)
-    label = models.TextField()
\ No newline at end of file
+    label = models.TextField()
+
+class RightsSupport(models.Model):
+
+    class Meta:
+
+        managed = False  # No database table creation or deletion  \
+                         # operations will be performed for this model.
+
+        permissions = (
```

Comment:
Same here I think we should add the additional permissions as we can restrict them to *_view for the main models.